frontend/class-opengraph.php

```diff
@@ -826,6 +829,22 @@ private function get_front_page_image() {
 	}
 
 	/**
+	 * Get the images of the posts page.
+	 */
+	private function get_posts_page_image() {
```

Comment:
Yeah, unfortunately that's the current convention within this class. Open Graph & Twitter classes need major cleanup and de-duplication, but it's not high priority at the moment.